### ISSUE TITLE
Third mouse wheel doesn't work correctly in ruffle.....

### ISSUE THREAD

**Issue by NewhereDigitalToys** [ruffle-rs/ruffle#5709]
### Describe the bug

![image](https://user-images.githubusercontent.com/75649061/143098365-2da83874-de1b-468b-b05f-9e8097ab3ee1.png)

Third mouse wheel doesn't work correctly in ruffle. The code below shows how it works correctly in Flash Player using ActionScript 2:

`var mouseListener:Object = new Object();
mouseListener.onMouseWheel = function(delta) {
	if (delta < 0) {
        nextFrame();
    } else if  (delta > 0) {
        prevFrame();
	}}
Mouse.addListener(mouseListener);`

What this code above does in AS2 is tell the Main Timeline, or any MovieClip you put the code in on the first Frame, to scroll through the timeline either Forward or Backward (as in: using the mouse wheel to scroll up to go forward in the timeline or scrolling down to go backward in the timeline). Another way to tweek this code to make it directly work specifically in a certain movie clip is:

`var mouseListener:Object = new Object();
mouseListener.onMouseWheel = function(delta) {
	if (delta < 0) {
        _root.MovieClip.nextFrame();
    } else if  (delta > 0) {
         _root.MovieClip.prevFrame();
	}}
Mouse.addListener(mouseListener);`

Note: This code also helps with using your finger on a smartphone, tablet or touchscreen to scroll up and down on webpages that use AS2. Hopefully in a future update of ruffle the DEV's will fix this by implementing this code setup in the ruffle JavaScript.

### Expected behavior

Everything is explained in "Describe the bug"

### Affected platform

Self-hosted version

### Operating system

Windows XP, Windows Vista, Windows 8, Windows 8.1, Windows 10, Android 8.1.0, Android 9, Android 11

### Browser

Latest versions of Mozilla Firefox, Google Chrome, Puffin Web Browser

**Comment by NewhereDigitalToys** [comment]
Here are the version examples: One that works good in flash player and one that don't work good in ruffle player. In flash player video, it plays all timelines right without glitching it by making all timelines move when you use the mouse wheel. But in the ruffle player, well....just see it for yourself on how it scrolls through all timelines when it is not suppose to.

https://user-images.githubusercontent.com/75649061/143398159-06f20680-8041-4ca7-9ead-9d416d78fa95.mp4

https://user-images.githubusercontent.com/75649061/143398214-87b07c51-b2c9-41b1-a468-ab4e0ec11496.mp4

**Comment by NewhereDigitalToys** [comment]
[MouseWheelScrollAS2.zip](https://github.com/ruffle-rs/ruffle/files/7647945/MouseWheelScrollAS2.zip)
Ok.....now, this is strange. I re-made a kinda rough sketch of how the code works in my Newhere Digital Toys website but for some reason now when i run this new .swf rough sketch (which is MouseWheelScrollAS2.zip) in the most recent ruffle demo on ruffle website as of now, the rough sketch seems to be working right compared to ruffle demo: Newhere Digital Toys website, where it does not work right.

Basically, the bug that happens on Newhere Digital Toys website on ruffle demo is that when you use the mouse wheel to scroll up or down it will move all timelines in .swf ruffle player which is wrong. But now in this re-made rough sketch version i made, it works good in demo because it only moves the animated yellow box and keeps the animated black/gray-ish box still. W.T.F.  ?

**Comment by kjarosh** [comment]
Hi! Is there still an issue? If yes, please include a SWF that works differently in Ruffle compared to FP and describe the differences.

**Comment by Lord-McSweeney** [comment]
Closing as no more information was provided. Please do reopen this issue if there's still a problem!
